src/tests/build.proj

```diff
@@ -543,6 +543,7 @@
   <Target Name="RestorePackage">
     <PropertyGroup>
       <_ConfigurationProperties>/p:TargetOS=$(TargetOS) /p:TargetArchitecture=$(TargetArchitecture) /p:Configuration=$(Configuration) /p:CrossBuild=$(CrossBuild)</_ConfigurationProperties>
+      <_ConfigurationProperties Condition="'$(UseLocalAppHostPack)' == 'true'">$(_ConfigurationProperties) -p:EnableAppHostPackDownload=false -p:EnableTargetingPackDownload=false -p:EnableRuntimePackDownload=false</_ConfigurationProperties>
```

Comment:
I guess the restore proj doesn't import [eng/targetingpacks.targets](https://github.com/dotnet/runtime/blob/main/eng/targetingpacks.targets)? If it does it's enough to pass down the option, right?

Reply:
Yup, this can be removed when `eng/targetingpacks.targets` is imported. I kept this stopgap PR small. (I will try importing it and open a separate PR)